spec/support/matchers/have_i18n_setting.rb

```diff
@@ -0,0 +1,17 @@
+require  'rspec/expectations'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.